extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientBuildConfig.java

```diff
@@ -88,6 +88,12 @@
     @ConfigItem
     public Optional<String> password;
 
+    /**
+     * Kubernetes oauth token
+     */
+    @ConfigItem
+    public Optional<String> token;
```

Comment:
Nice! :+1:

Reply:
thanks. yes that was missing for my tests ;)